[ONNX] Update exporter issue report instructions for quantized models (#113494)
pytorch/pytorch: docs/source/quantization.rst

   We don't have official GPU support yet, but this is an area of active development, you can find more information
   `here <https://github.com/pytorch/pytorch/issues/87395>`_

2. Where can I get ONNX support for my quantized model?

>    If you get errors exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository, prefix the issue title with ``[ONNX]``, and tag the issue as ``module: onnx``.

Author: ```suggestion
   If you get errors exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository. Prefix the issue title with ``[ONNX]`` and tag the issue as ``module: onnx``.
```

>    If you encounter an issue when exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository, prefix the issue title with ``[ONNX]``, and tag the issue as ``module: onnx``.

Author: ```suggestion
   If you get errorswhen exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository, prefix the issue title with ``[ONNX]``, and tag the issue as ``module: onnx``.
```

>    If you encounter an issue when exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository and prefix the issue title with ``[ONNX]`` and tag the issue as ``module: onnx``.

Author: ```suggestion
   If you encounter an issue when exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository, prefix the issue title with ``[ONNX]``, and tag the issue as ``module: onnx``.
```

>    If you encounter an issue with ONNX Runtime, open an issue at `GitHub - microsoft/onnxruntime <https://github.com/microsoft/onnxruntime/issues/>`_.

Author: ```suggestion
   If you encounter issues with ONNX Runtime, open an issue at `GitHub - microsoft/onnxruntime <https://github.com/microsoft/onnxruntime/issues/>`_.
```

   We don't have official GPU support yet, but this is an area of active development, you can find more information
   `here <https://github.com/pytorch/pytorch/issues/87395>`_

2. Where can I get ONNX support for my quantized model?

>    If you encounter an issue when exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository and prefix the issue title with ``[ONNX]``.

Author: ```suggestion
   If you encounter an issue when exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository and prefix the issue title with ``[ONNX]`` and tag the issue as ``module: onnx``.
```

>    If you encounter an issue when exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository by prefixing the issue title with ``[ONNX]``.

Author: ```suggestion
   If you encounter an issue when exporting the model (using APIs under ``torch.onnx``), you may open an issue in the PyTorch repository and prefix the issue title with ``[ONNX]``.
```

   If you encounter an issue with ONNX Runtime, open an issue at `GitHub - microsoft/onnxruntime <https://github.com/microsoft/onnxruntime/issues/>`_.

> 

Author: ```suggestion
```

>    You may reach out to people in this list: `PyTorch Governance | Maintainers | ONNX exporter <https://pytorch.org/docs/stable/community/persons_of_interest.html#onnx-exporter>`_

Reviewer: Instead of reaching out to maintainers would suggest telling people to open an issue and tag it as `module:onnx` - it's best not to have an email based workflow

Author: Thanks - done

>    You may reach out to people in this list: `PyTorch Governance | Maintainers | ONNX exporter <https://pytorch.org/docs/stable/community/persons_of_interest.html#onnx-exporter>`_

Author: ```suggestion
```
